### DISCUSSION TITLE
TDS - Tower Defense Strategy (Steam) Issues With UUVR

### DISCUSSION THREAD

**Discussion by BionicFreak** [Raicuparta/universal-unity-vr#30] [Q&A] [answered]
I noticed in Rai Pal that the Mono UUVR isn't an immediate option, like it is for other games, so does that mean it isn't compatible?
I also tried installing the mod manually, but then it caused the game to crash. So I just wanted to know, am I doing something wrong or is that version of Unity not compatible with UUVR?

Thanks in advance for any help!

**Comment by Raicuparta** [comment, marked as answer]
If Rai Pal detects a game as being the correct engine but doesn't show a certain mod (like UUVR), that means the mod isn't compatible. I'll make sure to make this more clear in future versions of Rai Pal.
